types/shared.ts

```diff
@@ -215,6 +215,8 @@ export enum GuildFeatures {
   PrivateThreads = "PRIVATE_THREADS",
   /** Guild is able to set role icons */
   RoleIcons = "ROLE_ICONS",
+  /** Guild has set up auto moderation rules */
+  AutoModeration = "ANIMATED_BANNER",
```

Comment:
What does this has to do with AutoModeration? type?

Reply:
Copy paste typo